homeassistant/components/google_maps/device_tracker.py

```diff
@@ -118,6 +118,16 @@ def _update_info(self, now=None):
                 )
                 _LOGGER.debug("%s < %s", last_seen, self._prev_seen[dev_id])
                 continue
+            if last_seen == self._prev_seen.get(dev_id, last_seen) and hasattr(
+                self, "success_init"
+            ):
+                _LOGGER.warning(
```

Comment:
`google_maps` is a polling integration, is it the case that there's sometimes no update between polls which is why the timestamp is the same?
If this is the case, this is not an error, and we should decrease log level to debug?

Reply:
definitely prefer debug. i just follow the original author